arch/arm/boot/dts/overlays/interludeaudio-analog-overlay.dts

```diff
@@ -0,0 +1,73 @@
+// Definitions for Interlude audio analog hat
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "brcm,bcm2835";
+
+	fragment@0 {
+		target = <&sound>;
+		__overlay__ {
+			compatible = "simple-audio-card";
+			i2s-controller = <&i2s>;
+			status = "okay";
+
+			simple-audio-card,name = "Interlude Audio Analog";
+
+			simple-audio-card,widgets =
+				"Line", "Line In",
+				"Line", "Line Out";
+
+			simple-audio-card,routing =
```

Comment:
Do we really want "Line out", to have 4 values?  We don't support differential.  What benefit does this provide the end user?

Reply:
those are audio routing paths for the DAPM dont believe the user gets to select those or make changes to them